cli/internal/runsummary/run_summary.go

```diff
@@ -54,14 +55,15 @@ type Meta struct {
 
 // RunSummary contains a summary of what happens in the `turbo run` command and why.
 type RunSummary struct {
-	ID                ksuid.KSUID        `json:"id"`
-	Version           string             `json:"version"`
-	TurboVersion      string             `json:"turboVersion"`
-	GlobalHashSummary *GlobalHashSummary `json:"globalCacheInputs"`
-	Packages          []string           `json:"packages"`
-	EnvMode           util.EnvMode       `json:"envMode"`
-	ExecutionSummary  *executionSummary  `json:"execution,omitempty"`
-	Tasks             []*TaskSummary     `json:"tasks"`
+	ID                ksuid.KSUID         `json:"id"`
+	Version           string              `json:"version"`
+	TurboVersion      string              `json:"turboVersion"`
+	GlobalHashSummary *GlobalHashSummary  `json:"globalCacheInputs"`
+	Packages          []string            `json:"packages"`
+	Pipeline          fs.PristinePipeline `json:"pipeline"`
```

Comment:
We don't need this here if it's just for printing in dry run

Reply:
Collapsing this conversation over into: https://github.com/vercel/turbo/pull/4545#discussion_r1178611355